spec/models/vacancy_spec.rb

```diff
@@ -566,6 +425,14 @@
       expect(vacancy.job_title).to eq(sanitized_title)
     end
 
+    it '#salary' do
```

Comment:
it does not read that well, I would use a context `#salary` and inside would put something like `it strips the tags`